infra/modules/doc-upload-s3/main.tf

```
@@ -0,0 +1,165 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+resource "aws_s3_bucket" "doc-upload" {
```

Aligning with making this module more generic, what do you think about this?

```suggestion
resource "aws_s3_bucket" "s3_encrypted" {
```

If this is ok with you, let's rename all these resources to `s3_encrypted`